app/lib/features/space/actions/activate_feature.dart

```diff
@@ -0,0 +1,280 @@
+import 'package:acter/common/providers/space_providers.dart';
```

Comment:
This action file code looks good in terms of generic function still facing bit difficulty to end-to-end flow. Would suggest to further break down and add some comment for more explanation.